service/resource/legacyv2/resource.go

```diff
@@ -180,22 +190,20 @@ type Resource struct {
 	pubKeyFile       string
 }
 
-type Event struct {
-	Type   string
-	Object *v1alpha1.AWSConfig
-}
-
 // NewUpdatePatch is called upon observed custom object change. It creates the
 // AWS resources for the cluster.
 func (s *Resource) NewUpdatePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*framework.Patch, error) {
+	s.logger.Log("info", "in NewUpdatePatch")
```

Comment:
Are these logs for debugging or will they be kept?

Reply:
Only for making clear the code flow during debug, removed.